src/router.ts

```diff
@@ -3,12 +3,15 @@ import Routes from './routes';
 import apiDocumentation from './routes/api-documentation';
 import articleList from './routes/article-list';
 import entryPoint from './routes/entry-point';
+import InMemoryArticles from './adaptors/in-memory-articles';
+import Articles from './adaptors/articles';
 
 export default (): Router => {
   const router = new Router();
+  const articles: Articles = new InMemoryArticles();
```

Comment:
Create this at the top level and pass it in.

See also https://github.com/libero/article-store/pull/60.